### PR TITLE
logout

### PR DESCRIPTION
**Pull request by dmitrig01**
@scotthasbrouck

### CODE REVIEW

File: src/3_branch.js [hunk 1]
```diff
@@ -464,7 +466,27 @@ Branch.prototype['setIdentity'] = wrap(callback_params.CALLBACK_ERR_DATA, functi
  */
 /*** +TOC_ITEM #logoutcallback &.logout()& ^ALL ***/
 Branch.prototype['logout'] = wrap(callback_params.CALLBACK_ERR, function(done) {
-	this._api(resources.logout, { }, done);
+	var self = this;
+	this._api(resources.logout, { }, function(err, data) {
+		if (err) { done(err); }
+
+		var data = {
```
Comment (scotthasbrouck) [inline src/3_branch.js]: couldn't we just reassign like `data = {...`
don't think we need to declare it since `data` is an argument in the callback

**Comment by scotthasbrouck** [comment]
This looks good, and works for me. I'll write some tests on this branch

File: src/3_branch.js (hunk 1 above)
Comment (dmitrig01) [inline src/3_branch.js]: oops yeah. we should probably actually name this variable something else then though. or just pass it in as a literal into session.set

**Comment by scotthasbrouck** [comment]
@dmitrig01 tested this manually, and wrote unit and integration tests and it looks good

**Comment by dmitrig01** [comment]
i'm guessing these failures are just sauce flukes?

**Comment by scotthasbrouck** [comment]
Yep, it's just sauce yet again.
All the tests pass locally

**Pull request merged**